javav2/example_code/cloudwatch/src/main/java/com/example/cloudwatch/CloudWatchScenario.java

```diff
@@ -0,0 +1,981 @@
+                break;
+            case "4":
+                selectedMetrics= metList.get(3);
+                break;
+            case "5":
+                selectedMetrics= metList.get(4);
+                break;
+            default:
+                System.out.println("You did not select a valid option.");
+                System.exit(1);
+        }
+        System.out.println("You selected "+selectedMetrics);
+        Dimension myDimension = getSpecificMet( cw, selectedNamespace);
+        System.out.println(DASHES);
+
+        System.out.println(DASHES);
+        System.out.println("3. Get statistics for the selected metric over the last day.");
+        String metricOption="";
+        ArrayList<String> statTypes = new ArrayList<>();
+        statTypes.add("SampleCount");
+        statTypes.add("Average");
+        statTypes.add("Sum");
+        statTypes.add("Minimum");
+        statTypes.add("Maximum");
+
+        for (int t=0; t<5; t++){
+            System.out.println("    " +(t+1) +". "+statTypes.get(t));
+        }
+        System.out.println("Select a metric statistic by entering a number from the preceding list:");
+        String myOption = reader.readLine();
+        switch (myOption) {
+            case "1":
+                metricOption= statTypes.get(0);
+                break;
+            case "2":
+                metricOption= statTypes.get(1);
+                break;
+            case "3":
+                metricOption= statTypes.get(2);
+                break;
+            case "4":
+                metricOption= statTypes.get(3);
+                break;
+            case "5":
+                metricOption= statTypes.get(4);
+                break;
+            default:
+                System.out.println("You did not select a valid option.");
+                System.exit(1);
+        }
+        System.out.println("You selected "+metricOption);
+        getAndDisplayMetricStatistics(cw, selectedNamespace, selectedMetrics, metricOption, myDate, myDimension);
+        System.out.println(DASHES);
+
+        System.out.println(DASHES);
+        System.out.println("4. Get CloudWatch estimated billing for the last week.");
+        getMetricStatistics(cw, costDateWeek);
+        System.out.println(DASHES);
+
+        System.out.println(DASHES);
+        System.out.println("5. Create a new CloudWatch dashboard with metrics.");
+        createDashboardWithMetrics(cw, dashboardName, dashboardJson);
+        System.out.println(DASHES);
+
+        System.out.println(DASHES);
+        System.out.println("6. List dashboards using a Paginator.");
+        listDashboards(cw);
+        System.out.println(DASHES);
+
+        System.out.println(DASHES);
+        System.out.println("7. Create a new custom metric by adding data to it.");
+        createNewCustomMetric(cw, dataPoint);
+        System.out.println(DASHES);
+
+        System.out.println(DASHES);
+        System.out.println("8. Add additional metric to the dashboard.");

```

Comment:
Should "Add additional metric" be "Add an additional metric" or "Add the additional metric" ?